From e00f789f64103bc094f382e671bfee9e57ceaed6 Mon Sep 17 00:00:00 2001
From: luojing <luojing@zhongguang.com>
Date: Thu, 18 Jun 2020 10:16:39 +0800
Subject: [PATCH] feat: add PA_SW node

---
 rau-fpga.dtsi | 24 ++++++++++++++++--------
 1 file changed, 16 insertions(+), 8 deletions(-)

diff --git a/rau-fpga.dtsi b/rau-fpga.dtsi
index f8b5c61..6fe4954 100755
--- a/rau-fpga.dtsi
+++ b/rau-fpga.dtsi
@@ -265,7 +265,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA0001040 0x40>,
-		      <0x0 0xA00014C0 0x04>; // Port_SW_Control
+		      <0x0 0xA00014C0 0x04>, // Port_SW_Control
+			  <0x0 0xA00014E0 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_d 3>, <&trx0_ad9371_d 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -278,7 +279,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA0001080 0x40>,
-		      <0x0 0xA00014C4 0x04>; // Port_SW_Control
+		      <0x0 0xA00014C4 0x04>, // Port_SW_Control
+			  <0x0 0xA00014E4 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_d 3>, <&trx0_ad9371_d 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -290,7 +292,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA00010C0 0x40>,
-		      <0x0 0xA00014C8 0x04>; // Port_SW_Control
+		      <0x0 0xA00014C8 0x04>, // Port_SW_Control
+			  <0x0 0xA00014E8 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_c 3>, <&trx0_ad9371_c 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -302,7 +305,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA0001100 0x40>,
-		      <0x0 0xA00014CC 0x04>; // Port_SW_Control
+		      <0x0 0xA00014CC 0x04>, // Port_SW_Control
+			  <0x0 0xA00014EC 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_c 3>, <&trx0_ad9371_c 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -317,7 +321,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA0001140 0x40>,
-		      <0x0 0xA00014D0 0x04>; // Port_SW_Control
+		      <0x0 0xA00014D0 0x04>, // Port_SW_Control
+			  <0x0 0xA00014F0 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_b 3>, <&trx0_ad9371_b 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -330,7 +335,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA0001180 0x40>,
-		      <0x0 0xA00014D4 0x04>; // Port_SW_Control
+		      <0x0 0xA00014D4 0x04>, // Port_SW_Control
+			  <0x0 0xA00014F4 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_b 3>, <&trx0_ad9371_b 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -342,7 +348,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA00011C0 0x40>,
-		      <0x0 0xA00014D8 0x04>; // Port_SW_Control
+		      <0x0 0xA00014D8 0x04>, // Port_SW_Control
+			  <0x0 0xA00014F8 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_a 3>, <&trx0_ad9371_a 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
@@ -354,7 +361,8 @@
 		#clock-cells = <1>;
 		compatible = "rau,rf-port-0.1";
 		reg = <0x0 0xA0001200 0x40>,
-		      <0x0 0xA00014DC 0x04>; // Port_SW_Control
+		      <0x0 0xA00014DC 0x04>, // Port_SW_Control
+			  <0x0 0xA00014FC 0x04>; // PA_SW
 
 		clocks = <&trx0_ad9371_a 3>, <&trx0_ad9371_a 4> ;
 		clock-names = "lo_rx_freq","lo_tx_freq";
-- 
2.7.4

